ci/plugins/cloudtest/hooks/post-command

```diff
@@ -17,6 +17,51 @@ kubectl() {
     bin/ci-builder run stable kubectl --context=kind-cloudtest "$@"
 }
 
+export_cov() {
+    bin/ci-builder run stable rust-cov export \
+      --ignore-filename-regex=.cargo/ \
+      --ignore-filename-regex=target/release/ \
+      --ignore-filename-regex=/cargo/ \
+      --ignore-filename-regex=/mnt/build/ \
+      --ignore-filename-regex=/rustc/ \
+      --format=lcov "$1" --instr-profile=coverage/"$BUILDKITE_JOB_ID".profdata src/ \
+      > coverage/"$BUILDKITE_JOB_ID"-"$(basename "$1")".lcov
+}
+
+if [ -n "${CI_COVERAGE_ENABLED:-}" ]; then
+    ci_uncollapsed_heading "cloudtest: Fetching binaries for coverage"
+    mkdir -p coverage/
+    chmod 777 coverage/
+    kubectl cp environmentd-0:/usr/local/bin/environmentd coverage/environmentd
+    kubectl cp "$(kubectl get pods -o name | grep -E 'cluster-' | head -n1 | sed -e "s#pod/##")":/usr/local/bin/clusterd coverage/clusterd
+    kubectl cp environmentd-0:/coverage coverage/
+    for pod in $(kubectl get pods -o name | grep -E 'cluster-'); do
+      kubectl cp "$pod":/coverage coverage/ || true # Could get deleted
```

Comment:
Just wondering: Is this another (parent) folder or why is it not necessary to remove the `pod/` prefix here?

Reply:
Actually, works with it, probably PEBCAK